virttest/libvirt_cgroup.py

```diff
@@ -275,6 +280,16 @@ def get_virsh_output_dict(self, vm_name=None, virsh_cmd=None):
             logging.error("There is no virsh cmd '%s'", virsh_cmd)
             return None
         result = func(vm_name, ignore_status=True)
+        return self.get_dict_output(result)
+
+    def get_dict_output(self, result):
```

Comment:
whether consider change get_dict_output to __get_dict_output since below there is another  example method: __get_dev_major_minor

Reply:
Thank you for your comment! I'd like to keep it because it'll be called in other tests in tp-libvirt directly.